test/lib/usd/utils/CMakeLists.txt

```diff
@@ -71,7 +71,7 @@ target_link_libraries(${TARGET_NAME}
         mayaUsdUtils
         tf
         sdf
-        ${Boost_PYTHON_LIBRARY} #Boost::python
+        Boost::python
```

Comment:
@robthebloke  Remove `Boost::python` or `${python_LIBRARIES}`. AFAIS, test_MayaTransformAPI test is written in C++, why are you linking against Python?

Remove `sdf, tf`. Why are you linking against these?

Reply:
Removing Tf, Boost::python, and PYTHON_LIBRARIES, results in unresolved externals.